extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -106,6 +107,7 @@ require([
 	'wikia.document',
 	'wikia.window'
 ], function (
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 8 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)